document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -491,7 +492,9 @@ public void drop() {
   static class PostgresResultIterator implements Iterator {
 
     private final ObjectMapper MAPPER = new ObjectMapper();
-    ResultSet resultSet;
+    private ResultSet resultSet;
+    private boolean didNext = false;
```

Comment:
can we rename this var?

Reply:
I have renamed it to `cursorMovedForward`